ElectronicsStore/PhotoCamera.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ElectronicsStore
+{
+    public abstract class PhotoCamera
+    {
+        protected string? ModelName;
+        protected decimal Price;
```

Comment:
PhotoCamera is also ElectronicDevice. So PhotoCamera  must be inherited ElectronicDevice. And maybe it's better to rename PhotoCamera to DeviceWithCamera. Because I reviewed it at the night and didn't think that mobile phone in real life is not a photo camera

Reply:
I am also confused, I hope it will be fine this time ...:)